custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html

```diff
@@ -116,10 +116,17 @@
                       Lactating Mothers enrolled for Anganwadi Services</a>
                 </div>
                 <div class="navigation-font pl-3 pr-3 mb-0"
+                     ng-if="haveAccessToFeatures==false"
                      ng-class="{'active-mobile-link': $location.path().indexOf('/demographics/adolescent_girls') !== -1}">
                     <a href="{$ goToStep('demographics/adolescent_girls', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
                       Adolescent Girls enrolled for Anganwadi Services</a>
                 </div>
+              <div class="navigation-font pl-3 pr-3 mb-0"
```

Comment:
Changes in mobile navigation
![nav_bar_mobile](https://user-images.githubusercontent.com/21310920/71279723-7fca3b00-237f-11ea-86aa-fc8025ac15b9.png)